src/components/notification/notification.ts

```diff
@@ -0,0 +1,66 @@
+import { unsafeCSS, html, LitElement } from 'lit';
+import { customElement, property, state } from 'lit/decorators.js';
+import { Control } from 'ol/control';
+import { unsafeSVG } from 'lit/directives/unsafe-svg.js';
+
+import style from '../../styles/notification.css?inline';
+import theme from '../../styles/theme.css?inline';
+
+import SVGCreator from '../../utils/svg-creator';
+import NotificationElement from '../../types/notification-element';
+
+@customElement('notification-box')
+class NotificationBox extends LitElement {
+  @property()
+  notification: NotificationElement = { type: '', message: '', rule: {} };
```

Comment:
I think the rule is probably not needed here and will be checked by another parent component. My guess is that the notification box will probably not be aware of zoom, layers, etc.
We probably need some specific controller or methods per rule

Reply:
Right, I remove from this component the rule.

I think the rules should be managed by the Control or other parent component. will see